### ISSUE TITLE
Create a solution script for general fisher questions

### ISSUE THREAD

**Issue by finnagin** [RTXteam/RTX#286]
Need to create a solution script for general fisher questions.

**Comment by dkoslicki** [comment]
After this has been tested thoroughly, please put an example in [QuestionExamples.py](https://github.com/RTXteam/RTX/blob/master/code/reasoningtool/QuestionAnswering/QuestionExamples.tsv).

**Comment by finnagin** [comment]
I did some testing on this and added an example question. It classified the question correctly every time I tested it.

**Comment by dkoslicki** [comment]
Excellent! Is it merged into master? Or are you waiting for the lists branch to be merge ready?

**Comment by finnagin** [comment]
Yeah fisher is merged into master. We are waiting on the list branch for obs/exp log ratio question.

**Comment by dkoslicki** [comment]
Excellent! So I'll close this issue out then, while you work on getting #285 merged.
